test/lib/jdk/test/lib/util/CoreUtils.java

```diff
@@ -152,6 +153,30 @@ public static String getCoreFileLocation(String crashOutputString) throws IOExce
                     line = line.trim();
                     System.out.println(line);
                     if (line.startsWith("|")) {
+                        if (line.split("\s", 2)[0].endsWith("systemd-coredump")) {
+                            // A systemd linux system. Try to retrieve core
+                            // file. It can take a few seconds for the system to
+                            // process the just produced core file so we may need to
+                            // retry a few times.
+                            System.out.println("Running systemd-coredump: trying coredumpctl command");
+                            String core = "core";
+                            try {
+                                for (int i = 0; i < 10; i++) {
+                                    Thread.sleep(5000);
+                                    OutputAnalyzer out = ProcessTools.executeProcess("coredumpctl", "dump",  "-1",  "-o", core, Long.valueOf(pid).toString());
```

Comment:
you can use `String::valueOf` to a string represenatation of `long`:
```suggestion
                                    OutputAnalyzer out = ProcessTools.executeProcess("coredumpctl", "dump",  "-1",  "-o", core, String.valueOf(pid));
```